app/screens/TransactionAuthorization.tsx

```diff
@@ -282,7 +278,7 @@ export function TransactionAuthorization (): JSX.Element | null {
           message={status === 'SIGNING' ? translate('screens/TransactionAuthorization', 'Signing...') : undefined}
         />
         {
-          (isSubmitted && attemptsRemaining !== undefined && attemptsRemaining !== MAX_PASSCODE_ATTEMPT) ? (
+          (attemptsRemaining !== undefined && attemptsRemaining !== MAX_PASSCODE_ATTEMPT) ? (
```

Comment:
I think the intention here is to show the error only after the at least one trial. However, the reminder should be there even no "new" attempt.

eg
Reach prompt UI, fail 3 times, cancel.
Try another transaction, user should see the reminder only 1 chance left, another failure lead to wallet backup wipe.

Reply:
Yep correct, but if that's the case, then we need to update the copy for it. Currently it has 'Incorrect Input' even without any inputs.

Reply:
Will check again, haven't seen that message.